cli/src/commands/init.rs

```diff
@@ -105,7 +111,7 @@ pub(crate) fn cmd_init(
                 tx.finish(ui)?;
             }
         }
-    } else if args.git {
+    } else if colocate {
```

Comment:
It creates non-colocated git repo. We'll need to initialize a `.git` repo first, then pass it to `init_external_git()`.